### PR TITLE
Create og_context_init function to determine that og_context is already running.

### PR DESCRIPTION
**Pull request by HelenaEksler**

### CODE REVIEW

File: og_context/og_context.module [hunk 1]
```diff
@@ -177,10 +177,29 @@ function og_context_og_invalidate_cache($gids = array()) {
   }
 }
 
+/**
+ * Determines if OG context has been already called.
+ * og_context function should not be called again until it is still running.
```
Comment (amitaibu) [inline og_context/og_context.module]: Line break missing.

Comment (amitaibu) [inline og_context/og_context.module]: actually this line isn't needed.

File: og_context/og_context.module [hunk 2]
```diff
@@ -177,10 +177,29 @@ function og_context_og_invalidate_cache($gids = array()) {
...
+ *
+ * @param bool $called
+ *  Determines if og_context function is still running.
```
Comment (amitaibu) [inline og_context/og_context.module]: `Determines if og_context() function has already been executed. Defaults to FALSE.`

File: og_context/og_context.module [hunk 3]
```diff
@@ -177,10 +177,29 @@ function og_context_og_invalidate_cache($gids = array()) {
...
+ *
+ * @return bool
+ *  Returns TRUE if function is still running.
```
Comment (amitaibu) [inline og_context/og_context.module]: `still running` isn't correct. You mean `was already executed`

File: og_context/og_context.module [hunk 4]
```diff
@@ -177,10 +177,29 @@ function og_context_og_invalidate_cache($gids = array()) {
   }
 }
 
+/**
+ * Determines if OG context has been already called.
+ * og_context() function should not be called again if it was already executed.
```
Comment (amitaibu) [inline og_context/og_context.module:182]: minor: there should be a line break after the first line.

**Comment by amitaibu** [comment]
I've fixed the minor comment and merged, thanks.

**Comment by pianomansam** [comment]
What is the purpose of og_context_init()? It never seems to do anything!

https://www.drupal.org/node/2713039

**Pull request closed without merging**